frontend/src/components/InfoBox/InfoBoxTags.js

```diff
@@ -169,8 +131,8 @@ class Tags extends Component {
                             y.section_id_normalized
                           )
                         )
-                        .map((a, i) => (
-                          <CourseLine key={i} data={a} />
+                        .map((data, i) => (
+                          <CourseDetails key={i} data={data} />
```

Comment:
Ideally key would not be the index, is there something unique in data that can be used?